ext/span.h

```diff
@@ -86,6 +87,7 @@ struct ddtrace_span_data {
     bool notify_user_req_end;
     struct ddtrace_span_data *next;
     struct ddtrace_root_span_data *root;
+    bool is_child_of_inferred_span;
```

Comment:
Why do we have this property at all? Isn't it redundant with child_root != NULL on the root_span_data?

Reply:
I see only one usage of it, and it's trivially replaced.